components/common/menu.tsx

```diff
@@ -1,16 +1,21 @@
 import Link from "next/link"
 import { useRouter } from 'next/router'
-import { faBars, faTimes } from "@fortawesome/free-solid-svg-icons"
+import { faBars, faCloud, faTimes, faUserPlus } from "@fortawesome/free-solid-svg-icons"
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
 
-const items = [
+const mainMenuItems = [
     { title: "Erstsemester", slug: "/erstiinfos/" },
     { title: "Wer sind wir?", slug: "/werwirsind/" },
     { title: "Wissenswertes", slug: "/faq/" },
     { title: "Sponsoring & Kooperation", slug: "/unternehmen/" },
     { title: "Kontakt", slug: "/kontakt/" },
 ]
 
+const userMenuItems = [
+    { title: 'Registrierung', link: 'https://registration.iwi-hka.de', icon: faUserPlus },
+    { title: 'Nextcloud', link: 'https://cloud.iwi-hka.de', icon: faCloud },
```

Comment:
We should consider swapping these items, since the registration is only needed once per user
```suggestion
    { title: 'Nextcloud', link: 'https://cloud.iwi-hka.de', icon: faCloud },
    { title: 'Registrierung', link: 'https://registration.iwi-hka.de', icon: faUserPlus },
```